src/menu/views/startup.c

```diff
@@ -9,6 +9,9 @@ static void draw (menu_t *menu, surface_t *d) {
 
 
 void view_startup_init (menu_t *menu) {
+    if (menu->settings.rom_autoload_enabled) {
+        //load_rom();
```

Comment:
set `menu->load_pending = true` and `menu->next_mode = MENU_MODE_LOAD_ROM` here

Reply:
and point to the correct ROM path in `menu->browser.directory` and `menu->browser.entry->name`

Reply:
Current attempt (working towards it) https://github.com/Polprzewodnikowy/N64FlashcartMenu/compare/develop...networkfusion:N64FlashcartMenu:rom-autoload-v2?expand=1